system_test/aest_sync_SUITE.erl

```diff
@@ -585,8 +585,8 @@ net_split_recovery(Cfg) ->
     T1 = erlang:system_time(millisecond),
 
     TargetHeight4 = MinedHeight3 + Length,
-    %% Wait for one extra block for resolving potential fork caused by nodes mining distinct blocks at the same time.
-    wait_for_value({height, 1 + TargetHeight4}, Nodes, (1 + Length) * ?MINING_TIMEOUT, Cfg),
+    %% Wait for extra blocks for resolving potential fork caused by nodes mining distinct blocks at the same time.
+    wait_for_value({height, 2 + TargetHeight4}, Nodes, (2 + Length) * ?MINING_TIMEOUT, Cfg),
```

Comment:
> You don't neccessarily have to wait extra here. It is sufficient to reach target height on all and then wait until they sync in the try_until loop below.

I drafted this in [this temporary commit](https://github.com/aeternity/epoch/commit/e9586b0c7025fe76c13d58b06164f8b69d1897d9) though it seems that the nodes config has ping interval 5s while the test thinks it is 120s. So if the ping interval in the test were 5s - as nodes - then 2 * ping interval might not be enough for mining a couple of blocks for resolving forks, leading to test instability. I am reviewing this.

Reply:
In #1609 I reviewed the value of the ping interval in the try-until loop, so that is actually of the same order of magnitude of the mining timeout (ping interval = 5s, mining timeout of 1 block = 3s). So relying on the try-until timeout to resolve the potential fork may make the test unstable.

I suggest to continue review on #1609 (once I remove its WIP label).